lib/http/client.rb

```diff
@@ -25,27 +25,32 @@ def initialize(default_options = {})
 
     # Make an HTTP request
     def request(verb, uri, opts = {}) # rubocop:disable Style/OptionHash
+      opts = @default_options.merge(opts)
+      req = prepare_request(verb, uri, opts)
+      res = perform(req, opts)
+      return res unless opts.follow
+
+      Redirector.new(opts.follow).perform(req, res) do |request|
+        perform(request, opts)
+      end
+    end
+
+    # Prepare an HTTP request
+    def prepare_request(verb, uri, opts = {}) # rubocop:disable Style/OptionHash
```

Comment:
Please rename this to `build_request`. IMO it would better reflect what method does.

Reply:
Sure, done.